javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,106 @@
+//freeCodeCamp user:@TatjanaOcedova
+
+//Flight booking fullname
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (firstname.length === 0 || surname.length === 0) {
+    return "Fullname is empty";
+  } else if (useFormalName == true) {
+    switch (gender) {
+      case "female":
+        return "Lady " + firstname + " " + surname;
+        break;
+      case "male":
+        return "Lord " + firstname + " " + surname;
+        break;
+      default:
+        return firstname + " " + surname;
+        break;
+    }
+  } else {
+    return firstname + " " + surname;
+  }
+}
+
+const fullname1 = getFullname("Tatjana", "Ocedova", true, "female");
+const fullname2 = getFullname("", "Ocedov", false, "male");
+
+console.log(fullname1);
+console.log(fullname2);
+
+//Event application
+const weekday = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+function getEventWeekday(daysLeft) {
+  const todaysDate = new Date();
+  const eventDate = new Date(
+    todaysDate.getFullYear(),
+    todaysDate.getMonth(),
+    todaysDate.getDate() + daysLeft
+  );
+  return weekday[eventDate.getDay()];
+}
```

Comment:
I think's this is a very clean solution. There is a hint that says you could look into using remainder operator, so you could think about how.
Hint: use remainder operator, array indexes and investigate new Date in js

Just a tip, you can also look into toLocaleDateString, to add localization to the dayname.